www/js/services.js

```diff
@@ -96,6 +96,39 @@ module.factory('Groups', function($http){
       }.bind(this));
     },
 
+    signUp: function(user, success){
+      //get reference to service in order to call other service
+      var self = this;
```

Comment:
I like the way you expose 'this' to the inner functions below using self.  Never seen it done this way before but makes sense and well thought out.  The more 'textbook'  way is to use the .bind function as we do above.  What the .bind(this) does is sets 'this' in the inner function to whatever you passed in.  @malc54 https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind